### ISSUE TITLE
[docs] Make user context available

### ISSUE THREAD

**Issue by brentvatne** [expo/expo#7333]
We should have the same user context that is available on expo.io available in the docs. This opens up the possibility to provide user-specific information and links.

**Comment by jonsamp** [comment]
bumping this one to keep it alive. @byCedric is working on a docs refactor at the moment. Either I or he can hook up the docs to have user data. Once we have that, I can work on designs to allow users to select an account and project that can then pre-fill code snippets with their account and project strings.

**Comment by byCedric** [comment]
Yep, but let's first get the code base into a maintainable state again! After that, we can add new features like user contexts 😄
